src/Sulu/Bundle/MediaBundle/Media/ListBuilderFactory/MediaListBuilderFactory.php

```diff
@@ -81,23 +81,25 @@ public function getListBuilder(
             $systemCollection = $this->collectionRepository
                 ->findCollectionByKey(SystemCollectionManagerInterface::COLLECTION_KEY);
 
-            $lftExpression = $listBuilder->createWhereExpression(
-                $fieldDescriptors['lft'],
-                $systemCollection->getLft(),
-                ListBuilderInterface::WHERE_COMPARATOR_LESS
-            );
-            $rgtExpression = $listBuilder->createWhereExpression(
-                $fieldDescriptors['rgt'],
-                $systemCollection->getRgt(),
-                ListBuilderInterface::WHERE_COMPARATOR_GREATER
-            );
+            if ($systemCollection instanceof CollectionRepositoryInterface) {
```

Comment:
same as above this more indicator that somebody failed to setup the fixtures correctly and usage of `Assert::notNull`